var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -347,3 +347,38 @@
 - command: systemctl disable google-accounts-daemon.service
   when:
     - platform == "gcp"
+
+#
+# We want the ssh service to start as early as possible during boot up,
+# so we explicitly remove all dependencies of the service here. We've
+# configured the service not to use the "default" dependencies, via the
+# "override.conf" file in the delphix-platform package. Unfortuantely
+# though, we can't remove any dependencies specified using "After=" in
+# the original SSH service file, by using that same "override.conf"
+# file. Thus, to remove the "After=" dependencies, we have to create a
```

Comment:
> we can't remove any dependencies specified using "After=" in the original SSH service file, by using that same "override.conf" file.

Can you elaborate on this? I thought you could typically clear parameters that takes lists as values by specifying the parameter again but without any value.

Reply:
I tried that. It didn’t work. The behavior I saw was that it concatenates all the “After” lines together. So adding “After=“ effectively does nothing. I couldn’t find docs to clearly explain how to clear it, either.

Reply:
OK, I guess that only works for some parameters. I see that **systemd.unit(5)** mentions
> Dependencies (After=, etc.) cannot be reset to an empty list, so dependencies can only be added in drop-ins. If you want to remove dependencies, you have to override the entire unit.